lib/const.rb

```diff
@@ -34,6 +35,7 @@
 CREW_DEST_DIR = CREW_BREW_DIR + 'dest'
 CREW_DEST_PREFIX = CREW_DEST_DIR + CREW_PREFIX
 CREW_DEST_LIB_PREFIX = CREW_DEST_DIR + CREW_LIB_PREFIX
+CREW_DEST_DLL_PREFIX = CREW_DEST_PREFIX + CREW_DLL_PREFIX
```

Comment:
If a variable is used in one package only, it should not be included in `const.rb`.

Reply:
it will be included in other packages later on, e.g., wine-mono and wine-gecko when I made them

Reply:
maybe we should add a `wine.rb` for wine specific variables?

Reply:
It's fine to add to const.rb.  I don't want to get too complicated where it becomes difficult to figure out what to do with specific packages.